src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs

```diff
@@ -39,7 +39,7 @@ public void TestAzureSqlGetContainers()
                 _logger, PsBackupProviderTypes.AzureSql, "Test-AzureSqlGetContainers");
         }
 
-        [Fact]
+        [Fact(Skip = "This workload is not supported anymore")]
```

Comment:
You need to remove these unsupported scenario tests, not skip them

Reply:
Can I take this is next release? We have a whole cleanup to do related to this workload